platformio.ini

```diff
@@ -378,6 +385,96 @@ extra_scripts =
     ${board:ttgo-t-display.extra_scripts}
     ${common:prog_ota.extra_scripts}
 
+
+; ********************************************************************************
+; TTGO T-Display S3 ESP32 WiFi and Bluetooth Module Development Board - Programming via USB
+; ********************************************************************************
+[env:ttgo-t-display-S3-usb]
+extends = board:ttgo-t-display-S3, common:prog_usb, mode:selected
+platform = espressif32
+build_flags =
+    ${board:ttgo-t-display-S3.build_flags}
+    ${mode:selected.build_flags}
+extra_scripts =
+    ${board:ttgo-t-display-S3.extra_scripts}
+    ${common:prog_usb.extra_scripts}
+
+; ********************************************************************************
+; TTGO T-Display ESP32 WiFi and Bluetooth Module Development Board
+; ********************************************************************************
+[board:ttgo-t-display-S3]
+extends = common:esp32_env, display:ttgo_tdisplay-S3
+board = esp32-s3-devkitc-1
+board_build.partitions = default_8MB.csv
```

Comment:
board_build.partitions could be skipped because it is already in the board configuration, right?

Reply:
Now needed to to 16MB custom partition usage.